articles/api/management/v2/tokens-flows.md

```diff
@@ -63,13 +62,13 @@ Long-lived tokens compromise your security. Following this process is <strong>NO
 
 You can use the [JWT.io Debugger](https://jwt.io/#debugger-io) to manually generate a token.
 
-The debugger allows you to edit the __Header__ and __Payload__ content on the right hand side of the screen. The token will automatically be updated on the left hand text area. Note that the debugger supports only `HS256` when editing header and payload.
+The debugger allows you to edit the __Header__ and __Payload__ content on the right-hand side of the screen. The token will automatically be updated on the left-hand text area. Note that the debugger supports only `HS256` when editing header and payload.
```

Comment:
Lets also remove this section on how to use JWT.io, all tokens should come from our server we should not tell customers to craft their own